File: pi_hadani.py
import random, math
vse = 50000000
#for j in range(5):
inCircle = 0
for i in range(vse):
    yCoord = random.random()
    xCoord = random.random()
    if ((xCoord-0.5)**2 +(yCoord -0.5)**2)**0.5 <= 0.5:
        inCircle = inCircle + 1
print((inCircle*4)/(vse))

File: pi_hadani.cs
double inC = 0;
double all = 50000000;
var rand = new Random();
double xCoord;
double yCoord;
for (int i = 0; i < all; i++)
{
    xCoord = rand.NextDouble();
    yCoord = rand.NextDouble();
    if (Math.Sqrt(Math.Pow(xCoord - 0.5,2)  + Math.Pow(yCoord - 0.5,2)) <= 0.5)
    {
        inC++;
    }
}
Console.WriteLine((inC * 4) / all);